_docs/ANDROID_SETUP.md

```diff
@@ -1,11 +1,36 @@
 # Grüvee Android Development Setup
 
+## Prerequisites
+
+### 🔥 Firebase
+
+#### Create and add `google-services.json` file to you project
+
+In Firebase console you need to create new application, get and add `google-services.json` file to the project:
+
+1. Head to https://console.firebase.google.com
+2. Go to `Project settings` under `settings icon` next to `Project Overview`
+3. Select `Android` under `General` tab, `Your apps` section
+4. On first step, fill in the fields for example:
+    - `Android package name`: com.gruvee
+    - `App nickname (optional)`: Gruvee
+5. On second step, click `Download google-services.json` button and then click `Next` button
+6. You can skip third and fourth step
+7. Add downloaded `google-services.json` to your `$GRUVEE_PROJECT/android/app` directory where `$GRUVEE_PROJECT` is the location of your root project.
+
+> Note: On **Windows 10** if it still sometimes says that it cannot find `google-services.json` file, restart the emulator and re-run `npm start` and `npm run android-start` commands.
+
 > **Please note that you need to start the emulator from Android Studio first before you run any commands.**
 >
 > For this just open up Android Studio and build the project, with an emulator set.
 >
 > To note: we have been developing on a Pixel 2 emulator and a Pixel 3 physical device, but feel free to use any device you'd like! At the end of the day it will be better for testing anyways.
 
+## Start Android Emulator
+
+1. After opening project in Android Studio for the first time you should see an event in `Event Log`. Open the `Event Log` and click `Configure` to automatically configure android project with `AndroidManifest.xml` files
+2. You can follow this guide https://developer.android.com/studio/run/managing-avds to setup new virtual device
+
```

Comment:
Hey out of curiosity, did you try just running the avd command on windows to start the emulator? Curious if this is just a Windows thing or not?

Reply:
Hmm, as I mentioned I have very little experience with Android Studio and emulators. It did not even cross my mind that it's possible to start avd from command line.
The line above it `For this just open up Android Studio and build the project, with an emulator set.` confused me a little bit as I didn't know exactly how to do it. And this is the way I found it to work 😂 But I assume there are "better" and more convenient options, right?

Reply:
Yup this is TOTALLY my fault as I guess I don't have this change here haha. The command I'm speaking of is:

`emulator -avd {AVDDeviceGoesHere}` (I have been using `Pixel_2_API_29`)

If this works, this will be awesome to use on all platforms imo

Reply:
I guess it's not specifically Windows thing. I will try running with the command and updated this part. Thanks for sharing it!